### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by Javert-ctrl**
This PR should contain the mutator, reader and GWAS_beta_VS_LTFH_transfored_beta functions, that does read and plot the analysis. The names can be better! Some vauge documentation has been written! 
I also made the function ask the user if they want a column with causal snps, if they have the beta column (in the mutator function).
This solves #32

### CODE REVIEW

**Comment by Javert-ctrl** [comment]
When Magnus' is back, we might want to change the vars factor in reader, if the lasso has a different output. (So we also obtain this in the dataset).

File: R/visualisations.R [hunk 1]
```diff
@@ -297,3 +297,61 @@ plot_pmgl_vs_true <- function(dataset,
...
+#' @param save_plot_path if \code{FALSE}, return the function returns a ggplot
+#' object. Else a path of the directory to save the plot to.
+#' @param plot_filename name of the file to be saved, including file extension.
+#'
+#' @return Either returns a \code{ggplot} object or saves the plot to
+#' \code{save_plot_path} and returns NULL.
+#'
+#' @export
+GWAS_beta_VS_LTFH_transfored_beta <- function(dataset,
+                                              transformed_beta,
```
Comment (RasmusBrostroem) [inline R/visualisations.R:319]: Is transformed_beta, linear_beta og bonferroni not columns in the dataset?

**Comment by jbirkesteen** [comment]
> When Magnus' is back, we might want to change the vars factor in reader, if the lasso has a different output. (So we also obtain this in the dataset).

Well Lasso is maybe a bit different from the others in a few ways other than just columns, since the output only gives the significant SNPs (I think). 
As I recall, there aren´t any p-values, but I'll ask @BigMag69 when he comes back tomorrow.

File: R/visualisations.R (hunk 1 above)
Comment (Javert-ctrl) [reply R/visualisations.R:319]: It should be after the mutator function has been applied :). The reason why I made it as input, is becouse you might want to compare other analysis types of beta-values. E.g the beta-values from liab agianst ltfh or linear gwas. But I think it is a better call to change the names of these inputs and the function ;).

**Comment by jbirkesteen** [comment]
It seems like you forgot to build the package prior to your push, @Javert-ctrl 🙂 
I'm a bit unsure of how the inputs for the plotting functions work. Maybe you have time later to meet on Discord?

**Review by jbirkesteen — APPROVED** [review]
Great 🚀 
I'll get started documenting it today. Documentation PR(documentation for everything) should be up by tomorrow noon at the latest, vignettes will also soon be up!

**Pull request merged**